Config/Inc/config.h

```diff
@@ -15,7 +15,12 @@ typedef enum ErrorStatus_e {ERROR = 0, SUCCESS = !ERROR} ErrorStatus;
 typedef enum SafetyStatus_e {SAFE = 0, DANGER = 1, OVERVOLTAGE = 2, UNDERVOLTAGE = 3} SafetyStatus;
 
 // NUCLEO or custom
+#ifdef CUSTOM_COMMAND_SIZE
+#define COMMAND_SIZE CUSTOM_COMMAND_SIZE
+#else
```

Comment:
@manthanand it might be a good time to delete the unused macros in config.h lol

Reply:
Lines 17-24 can be deleted.